platform/android/config.cmake

```diff
@@ -250,11 +250,6 @@ target_link_libraries(mapbox-gl
     PUBLIC -Wl,--gc-sections
 )
 
-# Create a stripped version of the library and copy it to the JNIDIR.
```

Comment:
```
before:
200,816,848 bytes: build/android-arm-v8-21/Release/libmapbox-gl.so
  5,510,144 bytes: platform/android/MapboxGLAndroidSDK/src/main/jniLibs/arm64-v8a/libmapbox-gl.so

after:
201,035,760 bytes: platform/android/MapboxGLAndroidSDK/build/intermediates/cmake/release/obj/arm64-v8a/libmapbox-gl.so
  5,510,144 bytes: platform/android/MapboxGLAndroidSDK/build/intermediates/bundles/default/jni/arm64-v8a/libmapbox-gl.so
```

The Gradle build automatically strips the symbols before putting them into the APK.

Given that the output path is now different, I also updated `metrics.sh`.